kdahyun/kdahyun.md

```diff
@@ -0,0 +1,12 @@
+# FirstName LastName (Unique ID)
+## Profile Pic: 
+## Call me by this name: dahyun
+## Cohort: July 2023
+## Email ID: gdh1685@kookmin.ac.kr
+## LinkedIn:
```

Comment:
I highly recommend one, if you explicitly do not want to create one then reply to this comment by saying, "I do not want to create my LinkedIn profile"